components/VerticalTabs/Pager.tsx

```diff
@@ -6,27 +8,51 @@ interface PagerProps {
 }
 
 const Pager = ({ identifier, count, current, hidden }: PagerProps) => {
+    const [isDesktop, setDesktop] = useState(false);
+
     const scrollTo = (idx: number) => {
         document
             .getElementById(`carousel-${identifier}-${idx}`)
             ?.scrollIntoView({ block: "center" });
     };
 
+    useEffect(() => {
+        if (window.innerWidth > 1450) {
+          setDesktop(true);
+        } else {
+          setDesktop(false);
+        }
+    
+        const updateMedia = () => {
+          if (window.innerWidth > 1450) {
+            setDesktop(true);
+          } else {
+            setDesktop(false);
+          }
+        };
+        window.addEventListener('resize', updateMedia);
+        return () => window.removeEventListener('resize', updateMedia);
+      }, []);
+    
+
     return (
         <div
             className={`flex items-center transition-opacity duration-500 lg:flex-col ${
                 hidden && "opacity-0"
             }`}
         >
             {[...Array(count)].map((_, i) => {
+                const dotSize = i === current ? (isDesktop ? "h-8" : "w-8") : (isDesktop ? "w-8" : "h-8")
```

Comment:
use the `xl` media query breakpoint (1280px). The docs I linked above has good examples